modular_citadel/code/modules/mob/cit_emotes.dm

```diff
@@ -198,15 +198,16 @@
 
 
 /datum/emote/living/mothsqueak
-	key = "msqueak"
-	key_third_person = "emits a tiny squeak"
-	message = "emits a tiny squeak!"
-	emote_type = EMOTE_AUDIBLE
+    key = "msqueak"
+    key_third_person = "emits a tiny squeak"
+    message = "emits a tiny squeak!"
+    emote_type = EMOTE_AUDIBLE
+    mob_type_allowed_typecache = list(/mob/living/carbon/)
 
 /datum/emote/living/mothsqueak/run_emote(mob/living/user, params)
-	if(ishuman(user))
-		if(user.nextsoundemote >= world.time)
-			return
-		user.nextsoundemote = world.time + 7
-		playsound(user, 'modular_citadel/sound/voice/mothsqueak.ogg', 50, 1, -1)
-	. = ..()
\ No newline at end of file
+    if(ishuman(user))
+        if(user.nextsoundemote >= world.time)
+            return
+        user.nextsoundemote = world.time + 7
+        playsound(user, 'modular_citadel/sound/voice/mothsqueak.ogg', 50, 1, -1)
+. = ..()
```

Comment:
This is not tabbed up. Needs a tab. or it'll compile not well.

Reply:
Should be fixed now